src/openrct2/ride/RideData.h

```diff
@@ -127,7 +128,8 @@ struct RideOperatingSettings
     uint8_t MaxBrakesSpeed;
     uint8_t PoweredLiftAcceleration;
     uint8_t BoosterAcceleration;
-    int8_t BoosterSpeedFactor;              // The factor to shift the raw booster speed with
+    int8_t BoosterSpeedFactor; // The factor to shift the raw booster speed with
+    uint16_t AccelerationFactor = 12;
     uint8_t OperatingSettingMultiplier = 1; // Used for the Ride window, cosmetic only.
```

Comment:
Since you put a value in-between, you will now have to modify all the RTDs that set `OperatingSettingMultiplier`. I’ll help you out: only the Twist sets this.